helm_deploy/prisoner-content-hub-frontend/templates/_helpers.tpl

```diff
@@ -73,7 +73,7 @@ Create external Kubernetes hostname
 {{- if .Values.ingress.tlsEnabled }}
 {{- $protocol = "https" }}
 {{- end }}
-{{- printf "%s://%s%s" $protocol .Values.ingress.domainPrefix (index .Values.ingress.hosts 0).suffix }}
+{{- printf "%s://siteName%s" $protocol (index .Values.ingress.hosts 0).suffix }}
```

Comment:
Up for discussion....

Reply:
I like the idea of renaming `domainPrefix` to `siteName`, that really makes the intent of that variable clearer 🙌 .

Looking at the code change, I'm probably reading it wrong (many hours into an assessment call), but I think that'll render `siteName` as a string, rather than interpolating the value.

Reply:
I wonder if we could print out the entire list of site URLs, that'd be a pretty nifty validation that of the live sites at the end of the deployment?

Reply:
`siteName` is replaced when creating an strategy, we can't do it at this stage as it's not known until a request is received.

Reply:
Like the last suggestion though 😄

Reply:
It feels a bit odd to be printing a URL that doesn't work here. I think I got tripped up because `siteName` isn't obviously a variable to be used elsewhere. 

I've added a comment on the point that does the replacement, and it feels like we can make that more elegant/explicit by using a different environment variable. I wonder if we'd use this output to set the `callbackUrl`? If not, we could just print the list of all hostnames, and add documentation around CI/helm, or address that other point?